packages/shared/src/hooks/squads/usePostToSquad.ts

```diff
@@ -149,10 +171,28 @@ export const usePostToSquad = ({
     [preview, displayToast, onSubmitLink, onPost, isPosting],
   );
 
+  const onUpdatePost = useCallback<UsePostToSquad['onUpdatePost']>(
+    (e, postId, sourceId, commentary) => {
+      e.preventDefault();
+
+      if (isUpdating) {
+        return null;
+      }
+
+      return updatePost({
```

Comment:
For some reason I get the generic "your post has been shared to squad" maybe we need a different callback here?
(also not sure it fires the same tracking perhaps?)